ext/drsyms/elfutils/elf_memory.c.patch

```diff
@@ -1,11 +0,0 @@
-diff --git a/libelf/elf_memory.c b/libelf/elf_memory.c
```

Comment:
Please augment the PR==merge description to mention we deliberately are not updating to 0.192 now and that when we do we will need to bring back the patch (may as well mention the exact culprit elfutils commit too).